utils/time.js

```diff
@@ -43,9 +43,22 @@ const getTimeInSecondAfter = ({ timestamp = Date.now(), days = 0, hours = 0, min
   return parseInt(timeInMilliseconds / 1000);
 };
 
+/**
+ * Returns time in seconds of timestamp before given duration
```

Comment:
are we returning time in `seconds` or `milliseconds` here

Reply:
It's actually a firestore timestamp returned from the function and contains both seconds and milliseconds as well. Let me correct the description here